php/php.editor/src/org/netbeans/modules/php/editor/completion/PHPCompletionItem.java

```diff
@@ -129,16 +131,22 @@ public abstract class PHPCompletionItem implements CompletionProposal {
     protected static final ImageIcon KEYWORD_ICON = IconsUtils.loadKeywordIcon();
     protected static final ImageIcon ENUM_CASE_ICON = IconsUtils.loadEnumCaseIcon();
     private static final int TYPE_NAME_MAX_LENGTH = Integer.getInteger("nb.php.editor.ccTypeNameMaxLength", 30); // NOI18N
-    final CompletionRequest request;
-    private final ElementHandle element;
-    private QualifiedNameKind generateAs;
     private static ScheduledExecutorService service = Executors.newSingleThreadScheduledExecutor();
     private static final Cache<FileObject, PhpLanguageProperties> PROPERTIES_CACHE
             = new Cache<>(new WeakHashMap<>());
+    private static final String AUTO_IMPORT_PARAM_FORMAT = "%s${php-auto-import default=\"\" fqName=%s aliasName=\"%s\" useType=%s editable=false}"; // NOI18N
+    private static volatile Boolean ADD_FIRST_CLASS_CALLABLE = null; // for unit tests
+
+    final CompletionRequest request;
+    private final ElementHandle element;
+    private QualifiedNameKind generateAs;
```

Comment:
Can we put non-final fields below the final ones?

Reply:
Will do that. Thanks!

Reply:
Fixed.